app/src/main/java/com/academy/ramon/marvelcomicviewer/util/HeroAdapter.java

```diff
@@ -16,28 +16,29 @@
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
+import de.hdodenhof.circleimageview.CircleImageView;
 
 /**
  * Created by Ramon on 12/21/2016.
  */
 
 public class HeroAdapter extends RecyclerView.Adapter<HeroAdapter.ViewHolder> {
 
+    private List<Heroes> heroesList;
+    private Context context;
```

Comment:
@Mathreyu why to hold this context reference since you are using parent.getContext()?